src/screens/ProfileScreen.tsx

```diff
@@ -136,7 +137,9 @@ export class ProfileScreen extends React.Component<Props, State> {
   }
 
   async componentDidMount() {
+    const { userId } = this.state
     this._initializeScreenData()
+    gaTrackPageView('/profile/' + userId, 'Profile Screen')
```

Comment:
Will this incorrectly log` /profile/undefined` if the user is not logged in giving us less useful insight ?

Reply:
Good catch. If the user goes to My Profile while not logged in, you are correct. I'll have it do /profile/user-not-logged-in so it is more specific.